src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -1,6 +1,6 @@
 <?php
-class Tribe__Tickets__Editor__Blocks__Rsvp
-extends Tribe__Editor__Blocks__Abstract {
+
+class Tribe__Tickets__Editor__Blocks__Rsvp extends Tribe__Editor__Blocks__Abstract {
```

Comment:
* :no_entry_sign: Missing doc comment for class Tribe__Tickets__Editor__Blocks__Rsvp
* :no_entry_sign: Class name is not valid; consider Tribe_Tickets_Editor_Blocks_Rsvp instead

<sub>hash: 1ce86266d216e8e844be00dd36a61f3b</sub>